### DISCUSSION TITLE
How to Change the Text in the Browser Tab

### DISCUSSION THREAD

**Discussion by alb3rtod** [chartbrew/chartbrew#238] [Q&A] [answered]
Is there any way to change the text in the browser tab? I mean the 'Chartbrew - Visualize your data by creating charts from your database' message in the browser tab. Is it possible? I would really like to see a shorter name in it.

![image](https://github.com/user-attachments/assets/5989c3df-d556-41c6-b71b-890e3529e5c9)

**Comment by razvanilin** [comment, marked as answer]
You can change the title from here: https://github.com/chartbrew/chartbrew/blob/master/client/index.html#L45

It's actually not the right title at the moment, so thanks for bringing it up. I think my AI copilot sneaked this one in

**Reply by alb3rtod** [reply]
Thank you so much my friend!
